tests/test_layout.py

```diff
@@ -299,6 +299,24 @@ def test_cycle_all_empty_frame(hlwm, delta):
     assert layout == hlwm.call('dump').stdout
 
 
+@pytest.mark.parametrize("floating_clients", [1, 2])
+@pytest.mark.parametrize("delta", [1, -1])
+def test_cycle_all_with_floating_clients(hlwm, delta, floating_clients):
```

Comment:
"all" is only floating clients here, right? Should we add non-floating parameterization, too, just to be sure?

Reply:
I think this is mostly covered because we have the empty frame here (the `None`). We can also add another testcase that mixes floating and tiling clients, but I'm not sure whether it tests something meaningful or just burns cpu time in the CI.

Reply:
I've added a new test case `test_cycle_all_traverses_tiling_and_floating` to cover multiple tiling and floating clients.

Reply:
Awesome, thank you!

Reply:
Thank you!